packages/identity/src/identity/webauthn.ts

```diff
@@ -112,7 +112,12 @@ async function _createCredential(
   }
 
   return {
-    ...creds,
+    // do _not_ use ...creds here, as creds is not enumerable in all cases
```

Comment:
@krpeacock: Isn't there any Typescript feature that would these types of issues (trying to spread non-enumerable objects)?

Reply:
Anything that extends a `class`, `Record` or `object` will inherit the `Iterable` trait. If the browser is not allowing us to spread fields from the creds response, there is an issue with the TypeScript-provided `dom` types